### ISSUE TITLE
Register Input Schema schema at SchemaStore, add permalink to location

### ISSUE THREAD

**Issue by fnesveda** [apify/apify-shared-js#401]
It would be nice to have the input schema registered at https://www.schemastore.org/json/, so that supporting editors like VSCode can offer autocompletion and validation.

It would also be nice to have a permalink to the input schema schema, which would redirect to the schema's location (currently at https://github.com/apify/apify-shared-js/blob/master/packages/input_schema/src/schema.json), so that we can put it to the input schema's `$schema` prop, and not worry about where the actual input schema schema file is located.

**Comment by B4nan** [comment]
> It would also be nice to have a permalink to the input schema schema

Can't we just use `https://raw.githubusercontent.com/apify/apify-shared-js/master/packages/input_schema/src/schema.json`? I guess that link won't change any time soon.

**Comment by fnesveda** [comment]
Yeah, but if it ever changes (e.g. we change the structure of the repo), everything using that link will break, that's why having a permalink that redirects to a correct location would be nice.

**Comment by JJetmar** [comment]
@fnesveda Maybe this might be closed as #597 was already merged?
